.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```
@@ -0,0 +1,143 @@
+name: 🐞 Bug Report
+description: File a bug report
+labels: ["bug", "needs-triage"]
+# assignees:
+#   - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there 👋,
+
+        Thank you for reporting the bug!
+
+        In case you want a new resource, data source or a new attribute supported in one of those,
+        please request it as a feature!
```

I dislike requiring fields and would rather use manipulative language to encourage it instead. 😈 

```suggestion
        Thank you for reporting the bug! Completing all sections of this form will help us find a quick solution.
        
        If the problem is a missing feature, then please use a [Feature Request](https://github.com/gitlabhq/terraform-provider-gitlab/issues/new?template=FEATURE_REQUEST.yml) instead.
```